Scripts/static-analyzer/Package.swift

```diff
@@ -0,0 +1,58 @@
+// swift-tools-version: 5.8
+
+import PackageDescription
+
+let package = Package(
+    name: "static-analyzer",
+    platforms: [.macOS(.v13)],
+    products: [
+        .executable(
+            name: "static-analyzer",
+            targets: ["static-analyzer"]
+        ),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/apple/swift-argument-parser", from: "1.2.0"),
+        .package(url: "https://github.com/apple/swift-syntax.git", exact: "508.0.0"),
+        .package(url: "https://github.com/apple/indexstore-db", revision: "swift-5.8-RELEASE"),
+        .package(url: "https://github.com/jpsim/SourceKitten.git", .upToNextMinor(from: "0.34.1")),
+        .package(url: "https://github.com/apple/swift-tools-support-core.git", from: "0.5.2"),
+    ],
+    targets: [
+        .executableTarget(
+            name: "static-analyzer",
```

Comment:
Is this name a bit too generic? 

Or... maybe this could be an encouragement to add more useful checks?

Reply:
Ah names... Yeah, I'm not sure what's an appropriate name for this project. My intention is adding more checks, i.e. finding `NSLocalizedString` call sites and checking if reverse domain string is used as the localization key.